packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/media/MediaRecorderBuilder.java

```diff
@@ -23,38 +23,73 @@ MediaRecorder makeMediaRecorder() {
   private final CamcorderProfile camcorderProfile;
   private final EncoderProfiles encoderProfiles;
   private final MediaRecorderFactory recorderFactory;
+  private final Integer fps;
+  private final Integer videoBitrate;
+  private final Integer audioBitrate;
 
   private boolean enableAudio;
   private int mediaOrientation;
 
   public MediaRecorderBuilder(
-      @NonNull CamcorderProfile camcorderProfile, @NonNull String outputFilePath) {
-    this(camcorderProfile, outputFilePath, new MediaRecorderFactory());
+      @NonNull CamcorderProfile camcorderProfile,
+      @NonNull String outputFilePath,
+      @NonNull Integer fps,
+      @NonNull Integer videoBitrate,
+      @NonNull Integer audioBitrate) {
```

Comment:
It seems like a lot of code is passing all three of these things around together; should they be part of some data class that groups settings?

Reply:
moved to data class